marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/MappingControllerTest.java

```diff
@@ -166,7 +166,7 @@ public void testMappingSettings() {
         // Check for defaults
         assertEquals("TestCustomerMapping", result.get("artifactName").asText());
         assertEquals(1, result.get("collections").size());
-        assertEquals("default-mapping", result.get("collections").get(0).asText());
+        assertEquals("Customer", result.get("collections").get(0).asText());
```

Comment:
Shouldnt the default collection be both entityType title and stepName to be consistent with pre 5.3?

Reply:
"default-mapping" is technically the step definition name, though I don't think it has any meaning other than to be a sensible default for a collection for mapped documents. If the entityType is used as the sensible default, then I don't see any reason nor meaning in the step definition name being used as a default collection too.

Reply:
I was looking for "TestCustomerMapping" and agree that "default-mapping" shouldnt be used as a default collection.

Reply:
@bsrikan added the step name to the default collections.